### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by damip**
closes https://github.com/massalabs/massa/issues/2876

and also updates the crypto libs and tokiio for better security and performance

### CODE REVIEW

File: massa-ledger-worker/src/ledger_db.rs [hunk 1]
```diff
@@ -340,12 +343,14 @@ impl LedgerDB {
         // datastore
         let mut opt = ReadOptions::default();
         opt.set_iterate_upper_bound(end_prefix(data_prefix!(addr)).unwrap());
-        for (key, _) in self.db.iterator_cf_opt(
+        for res in self.db.iterator_cf_opt(
             handle,
             opt,
             IteratorMode::From(data_prefix!(addr), Direction::Forward),
         ) {
```
Comment (adrien-zinger) [inline massa-ledger-worker/src/ledger_db.rs:350]: You can use `.flatten()` and process only the `Ok`, idem line 390.
Or should we break if Err?

**Review by adrien-zinger — APPROVED** [review]
LGTM! (with comment)

**Comment by damip** [comment]
bors merge

File: massa-ledger-worker/src/ledger_db.rs (hunk 1 above)
Comment (damip) [reply massa-ledger-worker/src/ledger_db.rs]: done, thanks !

**Pull request merged**